theia-electron/electron-builder/src/options.ts

```diff
@@ -0,0 +1,70 @@
+import { Configuration } from 'electron-builder';
+
+export function getTheiaBundleConfiguration(preview?: boolean): Configuration {
+    return {
+        electronDist: 'node_modules/electron/dist',
```

Comment:
Why is this required?

Reply:
`electronDist` tells `electron-builder` that it shouldn't pull binaries by itself, since we already have Electron installed inside `node_modules` thanks to `@theia/electron`. This distribution is also the one that is being patched by the scripts I added recently, so we really want to take full benefit from it.